src/targets/node/fetch.js

```diff
@@ -37,7 +37,15 @@ module.exports = function (source, options) {
 
   switch (source.postData.mimeType) {
     case 'application/x-www-form-urlencoded':
-      reqOpts.body = source.postData.paramsObj
+      code.unshift('const { URLSearchParams } = require(\'url\');')
+      code.push('const encodedParams = new URLSearchParams();')
+      code.blank()
+
+      source.postData.params.forEach(function (param) {
+        code.push('encodedParams.set(\'' + param.name + '\', \'' + param.value + '\');')
+      })
+
+      reqOpts.body = 'encodedParams.toString()'
```

Comment:
The `node-fetch` documentation and tests seems use pass the URLSearchParams object directly as the body, is there any particular reason for using `toString` instead?

Reply:
Seems that it [supports both URLSearchParams or a stringified body](https://github.github.io/fetch/#request-body), but I'll change it to just pass `encodedParams` directly.

Reply:
Tested locally and  without `.toString()` (and with) it works fine:

```js
const { URLSearchParams } = require('url');
const fetch = require('node-fetch');
const encodedParams = new URLSearchParams();

encodedParams.set('foo', 'bar');

let url = 'http://httpbin.org/post';

let options = {
  method: 'POST',
  qs: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'},
  headers: {
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded',
    cookie: 'foo=bar; bar=baz; '
  },
  body: encodedParams
};

fetch(url, options)
  .then(res => res.json())
  .then(json => console.log(json))
  .catch(err => console.error('error:' + err));

/*
{ args: {},
  data: '',
  files: {},
  form: { foo: 'bar' },
  headers:
   { Accept: 'application/json',
     'Accept-Encoding': 'gzip,deflate',
     'Content-Length': '7',
     'Content-Type': 'application/x-www-form-urlencoded',
     Cookie: 'foo=bar; bar=baz;',
     Host: 'httpbin.org',
     'User-Agent': 'node-fetch/1.0 (+https://github.com/bitinn/node-fetch)',
     'X-Amzn-Trace-Id': 'Root=1-5f90a62c-221fa1af7ec36f472ed54945' },
  json: null,
  origin: '98.37.58.48',
  url: 'http://httpbin.org/post' }
*/
```